pkgs/tools/networking/pihole-ftl/default.nix

```diff
@@ -0,0 +1,50 @@
+{ cmake
+, fetchFromGitHub
+, gmp
+, lib
```

Comment:
`lib` should be the first input in this list. Following that, the inputs should be sorted in order of usage below.